src/main/kotlin/no/skatteetaten/aurora/boober/controller/v1/InformationControllerV1.kt

```diff
@@ -0,0 +1,13 @@
+package no.skatteetaten.aurora.boober.controller.v1
+
+import no.skatteetaten.aurora.boober.controller.internal.Response
+import no.skatteetaten.aurora.boober.service.BitbucketService
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+class InformationControllerV1(val bitbucketService: BitbucketService) {
+
+    @GetMapping("/v1/auroraconfig")
```

Comment:
Etter min mening indikerer vanlig rest-semantikk at /v2/auroraconfig lister ut alle AuroraConfigs (ikke bare navnene på dem). Jeg syntes man bør kvalifisere dette med en requestvariabel, f.eks. fields som har verdien name. På denne måten *kan* man støtte utlisting av alle auroraconfigs på dette endepunktet senere (selv om vi nok aldri kommer til å gjøre det), men iallefall støtte /v1/auroraconfig/{name} som vi har snakket om tidligere uten at ressursene som returneres fra de to forespørselene er vesentlig forskjellig.

/v1/auroraconfig?fields=name (badrequest ved andre forespørseler nå i starten)